// [[Reason: undefined]] ////

//// [tests/cases/conformance/parser/ecmascript5/SkippedTokens/parserSkippedTokens16.ts] ////

===================================================================
--- TSC	declarations
+++ DTE	declarations
@@ -3,9 +3,9 @@
 //// [/.src/parserSkippedTokens16.d.ts]
 declare function Foo(): any;
 declare namespace M {
 }
-declare var x: any;
+declare var x: invalid;
 /// [Errors] ////
 
 parserSkippedTokens16.ts(1,1): error TS2552: Cannot find name 'foo'. Did you mean 'Foo'?
 parserSkippedTokens16.ts(1,6): error TS1005: ';' expected.
@@ -14,11 +14,12 @@
 parserSkippedTokens16.ts(2,27): error TS1127: Invalid character.
 parserSkippedTokens16.ts(3,3): error TS1109: Expression expected.
 parserSkippedTokens16.ts(6,5): error TS1138: Parameter declaration expected.
 parserSkippedTokens16.ts(8,14): error TS1109: Expression expected.
+parserSkippedTokens16.ts(8,14): error TS9007: Declaration emit for this file requires type resolution. An explicit type annotation may unblock declaration emit.
 
 
-==== parserSkippedTokens16.ts (8 errors) ====
+==== parserSkippedTokens16.ts (9 errors) ====
     foo(): Bar { }
     ~~~
 !!! error TS2552: Cannot find name 'foo'. Did you mean 'Foo'?
 !!! related TS2728 parserSkippedTokens16.ts:2:10: 'Foo' is declared here.
@@ -41,5 +42,7 @@
 !!! error TS1138: Parameter declaration expected.
     }
     var x       =
                  
-!!! error TS1109: Expression expected.
\ No newline at end of file
+!!! error TS1109: Expression expected.
+                 
+!!! error TS9007: Declaration emit for this file requires type resolution. An explicit type annotation may unblock declaration emit.
\ No newline at end of file
